### ISSUE TITLE
Low contrast on main page

### ISSUE THREAD

**Issue by edent** [sustainers/website#285]
![contrast](https://user-images.githubusercontent.com/837136/94807120-19327d00-03e7-11eb-9b87-758a103babb9.png)

The current text colour is `rgb(50, 198, 176)`. I think that's caused by this line setting the teal colour
https://github.com/sustainers/website/blob/master/_sass/_variables.scss#L3

I think the colour should be set to the (appropriately named) `$teal-dark-wcag: #128273;`  on 

https://github.com/sustainers/website/blob/master/_sass/components/_header-hero.scss#L38

There are a few other places where this needs to be changed. But I'll send a PR for this one first to make sure I don't break anything :-)
